src/entities/Publication.ts

```diff
@@ -0,0 +1,22 @@
+import { Field, ObjectType } from 'type-graphql';
+
+@ObjectType({ description: 'The Publications Model' })
+export default class Publication {
+  @Field()
+  slug: string;
+
+  @Field()
+  bio: string;
+
+  @Field()
+  feed: string;
+
+  @Field({ nullable: true })
+  img: string;
+
+  @Field()
+  name: string;
+
+  @Field()
+  url: string;
```

Comment:
if I were to be extra picky, I'd say this should be `websiteURL` to match the style from `Article.ts`

Reply:
Agreed, fixed